include/dds/core/xtypes/DynamicData.hpp

```diff
@@ -190,6 +190,28 @@ class WritableDynamicDataRef : public ReadableDynamicDataRef
         return *this;
     }
 
+    template<typename T, class = PrimitiveOrString<T>>
+    WritableDynamicDataRef& operator = (
+            const T& other)
+    {
+        value(other);
+        return *this;
+    }
+
+    WritableDynamicDataRef& operator = (
```

Comment:
This function is not needed since the `value()` function also can receive a string as a parameter.

Reply:
It's needed to allow receiving char* parameters through implicit conversions.